pkg/cluster/internal/create/actions/action.go

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package actions contains logic for steps to perform once an initial node
```

Comment:
I don't feel this lint is adding value, should we remove this linter @BenTheElder ?

Reply:
True.

As someone that started using kind mostly as a library to integrate into other functionality, I do appreciate having docstrings on most things. But these package level docs are really mostly useful if there is actual details to provide about the package. The ones missing these docstrings really don't need it, so there's not really much value to it.

I will exclude this check from the list. I think two of the linters both detect it, so just need to add the config for both to ignore it. But since it doesn't hurt anything, I'll leave the package docstrings added rather than trying to clean them all out.

Will probably not get to it until tomorrow or Monday. But with the ubuntu job failures, probably no rush until that's resolved anyway.

Reply:
we can switch the image , the newest ones does not have the bug

Reply:
+1

Reply:
it's less useful for internal packages but /shrug